src/components/LeftSideComponent.js

```diff
@@ -27,43 +31,38 @@ function LeftSideComponent() {
   }, [])
 
   return (
-    <>
-      <h4>Logged in users: {users.length}</h4>
-      {users.length > 0 ? (
-        <ul class={style.listGroup}>
-          {users.map((user) => {
-            return (
-              <li key={user.id} className={style.listGroupItem}>
-                {user.name}
-              </li>
-            )
-          })}
-        </ul>
-      ) : (
-        <p>No user online</p>
-      )}
-      <Nav
-        className='col-md-12 d-none d-md-block bg-light sidebar'
-        activeKey='/home'
-        onSelect={(selectedKey) => alert(`selected ${selectedKey}`)}
+    <div className='text-light'>
+      <Button onClick={handleShow} className='me-2' variant='warning' size='lg'>
+        Users
+      </Button>
+      <Offcanvas
+        show={show}
+        onHide={handleClose}
+        {...props}
+        className='text-light'
       >
-        <div className='sidebar-sticky'></div>
-        <Nav.Item>
-          <Nav.Link href='/home'>Active</Nav.Link>
-        </Nav.Item>
-        <Nav.Item>
-          <Nav.Link eventKey='link-1'>Link</Nav.Link>
-        </Nav.Item>
-        <Nav.Item>
-          <Nav.Link eventKey='link-2'>Link</Nav.Link>
-        </Nav.Item>
-        <Nav.Item>
-          <Nav.Link eventKey='disabled' disabled>
-            Disabled
-          </Nav.Link>
-        </Nav.Item>
-      </Nav>
-    </>
+        <Offcanvas.Header closeButton className='bg-secondary'>
+          <Offcanvas.Title className='bg-secondary'>
+            Users: {users.length}
+          </Offcanvas.Title>
+        </Offcanvas.Header>
+        <Offcanvas.Body className='bg-secondary'>
+          {users.length > 0 ? (
+            <ul class={style.listGroup}>
```

Comment:
```suggestion
            <ul className={style.listGroup}>
```
I got a warning on my console for this line, can you change it to `className`?